src/Components/Dropdowns.mdx

```diff
@@ -9,6 +9,18 @@ A type of menu that gives users the ability to select from multiple items or to
 
 Use this in place of other selection elements when the user needs to pick one option from a large set of options. Dropdown menus are appropriate inside forms, but can also be used on pages when a selection is required. (Example: a filter or sorting option)
 
+Install the component dependencies:
+
+`yarn add semantic-ui-react`
+
+`yarn add semantic-ui-css`
```

Comment:
The consuming library should not need to pull in `semantic-ui-css` as the css is provided as a dependency from this library. Does it work if we remove this dependency?